contents/blog/google-analytics-cost.md

```diff
@@ -0,0 +1,125 @@
+
+Thanks to its connection with Google, early entrance into the [web analytics](/web-analytics) market, and initial price tag (free), Google Analytics has long been the most popular product in its category. According to [BuiltWith](https://trends.builtwith.com/analytics/Google-Analytics), as of November 2024, it's used by 50% of the top million sites.
+
+Although Google Analytics is largely seen as a free product, this doesn't tell the full story. This post attempts to tell the rest of the story and help you figure out how much Google Analytics actually costs. 
+
+## Google Analytics 4 is free (but limited)
+
+[Google Analytics 4](/blog/posthog-vs-ga4) (GA4) is Google's latest iteration of analytics. It replaced Universal Analytics on July 1st, 2024 much to its users' chagrin. GA4 includes many standard features of analytics tools including:
+
+- Customizable capture and reporting of behavioral and demographic data
+- Traffic and acquisition analysis
+- Conversion tracking and funnels
+- User paths to track flows and drop-offs
+- Retention for users and revenue
+- Monetization analytics to track purchase value, LTV, and revenue metrics
+- UTM tracking and advertising analytics for marketing and Google Ads campaigns
+
+GA4 helps hundreds of thousands of entrepreneurs, marketers, developers, and analysts understand how users are using their site, how their marketing campaigns are performing, and figure out ways to make it better.
+
+One thing Google doesn't talk about is GA4's limitations. You can think of GA4 as the **free plan** of their analytics product. This means it has [limits](https://support.google.com/analytics/answer/11202874?sjid=1063875790934107320-NC) like:
+
+- Up to 500,000 sessions per report before sampling
+- Data retention up to 14 months
+- 25 event parameters per event
+- 25 user properties per property
+- 50 custom dimensions/metrics per property
+- 100 audiences
+- 200 explorations per user
+- BigQuery export is limited to 1M events daily
+
+While most GA users are under these limits, those getting close to the limit will likely need to upgrade to Google Analytics 360, the enterprise version of the product.
+
+### How and why is GA4 free?
+
+All businesses have costs, and as big as Google is, this is still true for them. There are multiple reasons why GA4 is free to use:
+
+1. The limits to data retention, size, and querying limit the most expensive potential costs they could face.
+
+2. By having strong integrations with Google Ads, GA4 encourages users to buy more of them. Ads have traditionally been Google's primary revenue driver, so other products can operate at losses.
+
+3. Your data is valuable to Google. It helps them improve ad targeting, product development, and data infrastructure among many other benefits. [GA has be declared illegal](/blog/is-google-analytics-illegal-microsite) in many countries because of this.
```

Comment:
```suggestion
3. Your data is valuable to Google. It helps them improve ad targeting, product development, and data infrastructure among many other benefits. [GA has been declared illegal](/blog/is-google-analytics-illegal-microsite) in many countries because of this.
```